package.json

```diff
@@ -67,10 +67,10 @@
     "simple-git": "1.65.0"
   },
   "scripts": {
-    "build": "babel src --out-dir dist && echo '\n\\033[0;34m[+] \\033[0;32mBuild done\\033[0m'",
-    "build:watch": "onchange 'src/**/*.js' 'node_modules/forest-express/dist/*' --no-exclude -i -- yarn build",
+    "build": "babel src --out-dir dist",
+    "build:watch": "onchange 'src/**/*.js' 'node_modules/forest-express/dist/*' --no-exclude -i -- babel --source-maps inline --out-dir dist src",
     "lint": "./node_modules/eslint/bin/eslint.js .eslint-bin src test",
     "test": "jest",
     "test:coverage": "jest --coverage"
   }
-}
+}
```

Comment:
[Editor configuration] No reason to have a change for this line. I guess you should configure you editor to have a new line at the end of files on save.